### PR TITLE
fix: stats layout and add tooltips

### PR DESCRIPTION
**Pull request by theflucs**
Fix responsive card layout
Add tooltips to pr and contribution title (removed title tag), so fixing the [ issue #95](https://github.com/DevLeonardoCommunity/github-stats/issues/95)

### CODE REVIEW

File: src/components/RepositoryContributionsCard.tsx [hunk 1]
```diff
@@ -13,58 +13,86 @@ export const RepositoryContributionsCard = ({
...
+              (
+                { pullRequest: { state, title, id, url } }: any,
+                index: number
+              ) => (
+                <div
+                  key={id}
+                  className={`flex items-center justify-between gap-2 tooltip text-left ${
+                    index === 0 ? "tooltip-bottom" : "tooltip-top"
+                  }`}
+                  data-tip={title}
```
Comment (theflucs) [inline src/components/RepositoryContributionsCard.tsx:65]: This is a workaround for correctly displaying the tooltips.
If it is set to` tooltip-top` the first item's tooltip doesn't show, as well as if it is set to  `tooltip-bottom` the last one doesn't appear either. Feel free to discard these lines if you know a cleaner way to achieve the correct displaying of the tooltips.

File: src/components/RepositoryContributionsCard.tsx [hunk 2]
```diff
@@ -13,58 +13,81 @@ export const RepositoryContributionsCard = ({
...
+                  {totalCount}
+                </div>
+              </div>
             </div>
-          ))}
+          </div>
+          <div className="max-h-[22rem] hide-scrollbar overflow-auto flex flex-col gap-1">
+            {nodes?.map(
+              (
+                { pullRequest: { state, title, id, url } }: any,
```
Comment (Balastrong) [inline src/components/RepositoryContributionsCard.tsx:57]: We should probably add a better type here than `any`, I think there's already a type in `types/github.ts` for that

**Review by Balastrong — CHANGES_REQUESTED** [review]

**Review by CBID2 — CHANGES_REQUESTED** [review]
For the most part, it looks good, but I do see some of the long titles not be in-line (see the card that says "Accessible For All" for reference in the below screenshot) 
test for tooltip contribution.

Is there a way you can make the tooltip work for all long titled-PRs @theflucs? :) What do you think @Balastrong?

File: src/components/RepositoryContributionsCard.tsx (hunk 1 above)
Comment (theflucs) [reply src/components/RepositoryContributionsCard.tsx:65]: Ouch, I didn't notice it. And it makes sense it doesn't work. We need to find another way :thinking:
@CBID2 you can take over this if you like

**Comment by theflucs** [comment]
@CBID2 are you sure your code is aligned with this branch? The PR title shows the 3 dots, if too long, and also the contribution icons should be aligned with the PR title pill, they don't look like. The tooltip part I think is fixed for the PR title but not for the contribution title, as mentioned above by @Balastrong

**Comment by CBID2** [comment]
> @CBID2 are you sure your code is aligned with this branch? The PR title shows the 3 dots, if too long, and also the contribution icons on the right should be aligned with the PR title pill, they don't look like. The tooltip part I think is fixed for the PR title but not for the contribution title, as mentioned above by @Balastrong. You can have a look at it and see if you come up with a solution

@Balastrong this deployment needs approval again.

**Comment by Balastrong** [comment]
@CBID2 the deploy is not working at the moment, it redirects to an old version.

In order to test this branch you should checkout it locally and run the app there :)

**Review by Balastrong — APPROVED** [review]
Overall it looks good to me as the scope of the PR was only on PR titles actually, I'd go with merging it so we keep small changes.

@CBID2 thanks for the review! If you find something else that needs to be tweaked feel free to create new issues so we can start from there!

**Comment by CBID2** [comment]
> @all-contributors please add @CBID2 for review

@Balastrong, you used my name instead of @theflucs. Did you mean to do that?

**Comment by Balastrong** [comment]
Yes, I added you for the review :D

**Pull request merged**